Test_oM/UnitTests/UnitTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*	
+ * This file is part of the Buildings and Habitats object Model (BHoM)	
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.	
+ *	
+ * Each contributor holds copyright over their respective contributions.	
+ * The project versioning (Git) records all such contribution source information.	
+ *                                           	
+ *                                                                              	
+ * The BHoM is free software: you can redistribute it and/or modify         	
+ * it under the terms of the GNU Lesser General Public License as published by  	
+ * the Free Software Foundation, either version 3.0 of the License, or          	
+ * (at your option) any later version.                                          	
+ *                                                                              	
+ * The BHoM is distributed in the hope that it will be useful,              	
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               	
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 	
+ * GNU Lesser General Public License for more details.                          	
+ *                                                                            	
+ * You should have received a copy of the GNU Lesser General Public License     	
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      	
+ */
+
+using BH.oM.Base;
+using System.Collections.Generic;
+using System.Reflection;
+using System.ComponentModel;
+
+namespace BH.oM.Test.UnitTests
+{
+    [Description("Defines a datadriven unit test for a specific method. Class defines inputs, method to be run and expected output.")]
```

Comment:
I think this should either be `data-driven` or `data driven` - but it's definitely not one word. Also, the class itself does not `defines inputs, method to be run, and expected output` - it only defines the method, and the test data to be used (which then defines the inputs and expected output). I think it's potentially misleading to say this class defines them, if that makes sense?

Reply:
Yes - I would go with data-driven.

Reply:
> Also, the class itself does not defines inputs, method to be run, and expected output - it only defines the method, and the test data to be used (which then defines the inputs and expected output). I think it's potentially misleading to say this class defines them, if that makes sense?

Is semantics - but in terms of clarity - I might go with @IsakNaslundBh's original phrase _"Class defines inputs, method to be run and expected output."_ 

OK it defines the inputs and outputs through a sub property object, but it is a very clear description of the purpose of the object.
What do we think? 😄